io.museeks.app.yaml

```diff
@@ -0,0 +1,49 @@
+id: io.museeks.app
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: museeks
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --share=network
+  - --socket=pulseaudio
+  - --env=WEBKIT_DISABLE_COMPOSITING_MODE=1
+
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/martpie/museeks/releases/download/0.22.0/Museeks_0.22.0_amd64.deb # make that 0.22
```

Comment:
so what happen on aarch64?

Also why no build from source?

Reply:
Museeks does not support aarch64 builds, working on it but it's a bit complicated to support 3 platforms * 2 arch * 3-4 formats.

[martpie/museed#784](https://github.com/martpie/museeks/issues/784), [martpie/museed#850](https://github.com/martpie/museeks/pull/850)